src/utils/insert.test.ts

```diff
@@ -54,11 +54,16 @@ describe('insert', () => {
   });
 
   it('should insert null as value when value to be inserted is falsy', () => {
```

Comment:
Should you change this word ?

```suggestion
  it('should insert undefined as value when value to be inserted is falsy', () => {
```

Reply:
yep!